SwaggerGen/Swagger/Type/AllOfType.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace SwaggerGen\Swagger\Type;
+
+/**
+ * allOf compound type
+ *
+ * @package    SwaggerGen
+ * @author     Bruce Weirdan <weirdan@gmail.com>
+ * @copyright  2014-2015 Martijn van der Lee
+ * @license    https://opensource.org/licenses/MIT MIT
+ */
+class AllOfType extends AbstractType
+{
+	private $allOfItems = array();
+	private $mostRecentItem;
+	protected function parseDefinition($definition)
+	{
+		$pattern = self::REGEX_START . 'allOf' . self::REGEX_CONTENT . self::REGEX_END;
```

Comment:
Is it necessary for `allOf` to be case sensitive?
If it has to be case sensitive, I'd prefer all lower-case.

Reply:
You mean as a part of the pattern? Yeah, that could be lowercase.